### DISCUSSION TITLE
Sample github action to lint i18n

### DISCUSSION THREAD

**Discussion by corentinmusard** [amannn/next-intl#1206] [Ideas]
Is there a way to lint next-intl in a CI/CD environment?

I use vscode extension `i18n Ally` and I want to get a similar check as part of a CI pipeline:

- CI fails if some keys are unused
- CI fails if some keys are missing

**Comment by welfoz** [comment]
Yeahh, it would be so helpful! 
How did you solve this?

**Reply by corentinmusard** [reply]
I did not solve it. I'm just being extra careful when refactoring code

Btw, now that Biome v2 has plugin and multi-file analysis support, it might be interesting to build those lint rules within biome

**Comment by amannn** [comment]
This is closely related: [Validating messages](https://next-intl.dev/docs/workflows/messages)
